cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-rococo/src/tests/asset_transfers.rs

```diff
@@ -15,27 +15,7 @@
 
 use crate::*;
 
-#[test]
-fn send_rocs_from_asset_hub_rococo_to_asset_hub_westend() {
-	let prefund_amount = 10_000_000_000_000u128;
-	let owner: AccountId = AssetHubWestend::account_id_of(ALICE);
-	let roc_at_asset_hub_westend =
-		MultiLocation { parents: 2, interior: X1(GlobalConsensus(NetworkId::Rococo)) };
-	AssetHubWestend::force_create_foreign_asset(
-		roc_at_asset_hub_westend,
-		owner,
-		true,
-		ASSET_MIN_BALANCE,
-		vec![],
-	);
-
-	let sender_balance_before =
-		<AssetHubRococo as Chain>::account_data_of(AssetHubRococoSender::get()).free;
-	let receiver_rocs_before = AssetHubWestend::execute_with(|| {
-		type Assets = <AssetHubWestend as AssetHubWestendPallet>::ForeignAssets;
-		<Assets as Inspect<_>>::balance(roc_at_asset_hub_westend, &AssetHubWestendReceiver::get())
-	});
-
+fn send_asset_from_asset_hub_rococo_to_asset_hub_westend(id: MultiLocation, amount: u128) {
```

Comment:
@acatangiu I dont know if this is possible to deduplicate easily:
```
fn send_asset_from_asset_hub_rococo_to_asset_hub_westend
fn send_asset_from_asset_hub_westend_to_asset_hub_rococo
```
to the one generic
```
fn send_asset_from_asset_hub_to_asset_hub
```

they are doing exactly the same thing AH1 -> BH1 -> BH2 -> AH2 and AH2 -> BH2 -> BH1 -> AH1,
we could possibly reuse for BHKusama/BHPolkadot,
maybe with some macro helper? But sure, also no that big issue to have copies.

but very cool that we have bi-directional tests now 👍 👍 👍